src/poliastro/bodies.py

```diff
@@ -58,10 +58,10 @@ def _rot_elements_at_epoch(T, d):
 
 
 class Body(_Body):
-    """Class to represent a generic body of the Solar System.
+    """Class to represent a generic body.
 
     """
-    def __init__(self, parent, k, name, symbol=None, R=0 * u.km):
+    def __init__(self, parent, k, name, symbol=None, R=0 * u.km, **kwargs):
```

Comment:
Can you think of any use case for the extra `**kwargs`?

Reply:
Any time you'd like to put extra data on a Body you can pass it as a keyword argument and use it from kwargs.  This way you won't need to add some attribute after creation or subclass body to add your own parameters.

Reply:
Sure, but this applies to any kind of object. What kind of extra parameters would you add to Bodies that are not in the constructor?

Reply:
Suppose you wanted to add a comment attribute to each body or some kind of additional value you are using for a custom calculation.  It would be easier to allow people to pass in any keywords they want rather than add a bunch of new parameters for whatever use case may come along?

Reply:
Fair enough, let's leave it like this then :)